File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/ScmI18Variable.java
package com.tencent.devops.scm.api.pojo;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * scm i18变量,只声明code,118转换在bk-ci工程，运行时转换
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ScmI18Variable {

    // i18 code
    private String code;
    // 转换参数
    private List<String> params;
    private String defaultMessage;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/ScmI18Variable.kt
package com.tencent.devops.scm.api.pojo

/**
 * scm i18变量,只声明code,118转换在bk-ci工程，运行时转换
 */
data class ScmI18Variable(
    // i18 code
    val code: String,
    // 转换参数
    val params: List<String>? = null,
    // 默认消息
    val defaultMessage: String? = null
)
